test/kennel/models/monitor_test.rb

```diff
@@ -581,6 +580,13 @@ def diff_resource(e, a)
       diff_resource({ type: -> { "query alert" } }, {}).must_equal []
     end
 
+    describe "#locked" do
+      it "ignores deprecated option locked" do
+        expected_basic_json[:options].delete(:locked)
```

Comment:
seems sus ... it's already removed from expected_basic_json above ?

Reply:
added a commit to address this 👍